core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreMainActivity.kt

```diff
@@ -109,6 +110,22 @@ abstract class CoreMainActivity : BaseActivity(), WebViewProvider {
     if (destination.id !in topLevelDestinations) {
       handleDrawerOnNavigation()
     }
+    if (destination.id == readerFragmentResId) {
+      unLockReaderNavView()
```

Comment:
unlock is 1 word and my specific naming for the drawer was supposed to be a hint  about how readerDrawerNavView could be renamed

Reply:
@macgills What should we rename it?

Reply:
D'oh! we don't need that name anymore.